### PR TITLE
Add support for Terraform 0.12

### LINKED ISSUE #20: Add support for Terraform 0.12

### LINKED ISSUE #24: Change terraform-google-log-export reference after TF0.12 release
This PR:
https://github.com/terraform-google-modules/terraform-google-event-function/pull/23

References @aaron-lane TF 0.12 branch, it should be updated after new version is released.

### PR DESCRIPTION
**Pull request by bohdanyurov-gl**
Fixes #20
Fixed #24

### CODE REVIEW

**Comment by bohdanyurov-gl** [comment]
image

**Comment by bohdanyurov-gl** [comment]
image

**Comment by bohdanyurov-gl** [comment]
Relevant CI fix: https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/pull/251

File: test/integration.sh [hunk 1]
```diff
@@ -40,6 +40,15 @@ setup_environment() {
   export TF_VAR_project_id="${PROJECT_ID}"
   export TF_VAR_region="${REGION:-us-east1}"
   export TF_VAR_zone="${ZONE:-us-east1-b}"
+
+  # Stubs for module/*
```
Comment (aaron-lane) [inline test/integration.sh]: If these values are required then they should be provided by the pipeline or the developer, not stubbed here.

**Pull request merged**